js/core/gameSim.js

```diff
@@ -120,20 +121,16 @@ define(["lib/underscore", "util/helpers", "util/random"], function (_, helpers,
         var out, p, t;
 
         // Simulate the game up to the end of regulation
-        this.simPossessions();
+        this.simRegulation();
 
         // Play overtime periods if necessary
         while (this.team[0].stat.pts === this.team[1].stat.pts) {
-            if (this.overtimes === 0) {
-                this.numPossessions = Math.round(this.numPossessions * 5 / 48);  // 5 minutes of possessions
-                this.dt = 5 / (2 * this.numPossessions);
-            }
             this.t = 5;
```

Comment:
Move this and the next 4 lines inside simOvertime, seems more organized that way.